components/molecules/sidebar/__tests__/stories.js

```diff
@@ -23,7 +25,34 @@ export default {
     readme: {
       sidebar: Readme
     }
+  },
+  argTypes: {
+    backgroundColor: { control: 'color' }
   }
 }
 
-export const main = () => <Sidebar data={Data} />
+const BaseComponent = (props = {}) => {
+  const { Brand } = useContext(ConfigContext)
+
+  const defaultProps = {
+    brand: Brand.logo,
+    data: Data,
+    ...props
+  }
+
+  return <Sidebar {...defaultProps} />
+}
```

Comment:
@esuarezz Spacing in code is nice - seperate blocks of code its usually easier to read.